pandas/io/formats/style.py

```diff
@@ -1589,6 +1678,150 @@ def pipe(self, func: Callable, *args, **kwargs):
         return com.pipe(self, func, *args, **kwargs)
 
 
+class _Tooltips:
+    """
+    An extension to ``Styler`` that allows for and manipulates tooltips on hover
+    of table data-cells in the HTML result.
+
+    Parameters
+    ----------
+    css_name: str, default "pd-t"
+        Name of the CSS class that controls visualisation of tooltips.
+    css_props: list-like, default; see Notes
+        List of (attr, value) tuples defining properties of the CSS class.
+    tooltips: DataFrame, default empty
+        DataFrame of strings aligned with underlying ``Styler`` data for tooltip
+        display.
+
+    Notes
+    -----
+    The default properties for the tooltip CSS class are:
+
+        - visibility: hidden
+        - position: absolute
+        - z-index: 1
+        - background-color: black
+        - color: white
+        - transform: translate(-20px, -20px)
+
+    Hidden visibility is a key prerequisite to the hover functionality, and should
+    always be included in any manual properties specification.
+    """
+
+    def __init__(
+        self,
+        css_props: List[Tuple[str, Union[str, int, float]]] = [
```

Comment:
Does this need to be a list? Would be preferable to have a non-mutable default argument

Reply:
Same reason as above: List is for consistency with with underlying `Styler` `table_styles`.

Again shall I change? I probably prefer as is.. fits with existing documentation and other Styler methods.

Reply:
Hmm fair enough. I'm being pedantic but given your callout let's leave as is and in the future might want to comprehensively change these to non-mutable default arguments